src/lib/components/LoginModal.svelte

```diff
@@ -46,9 +46,8 @@
 			{:else}
 				<button
 					class="flex w-full items-center justify-center whitespace-nowrap rounded-full border-2 border-black bg-black px-5 py-2 text-lg font-semibold text-gray-100 transition-colors hover:bg-gray-900"
-					on:click={(e) => {
+					on:click|preventDefault={() => {
 						if (!cookiesAreEnabled()) {
-							e.preventDefault();
```

Comment:
this line 
```js
if (!cookiesAreEnabled()) {
   e.preventDefault();
```
was replaced to using svelte modifier `on:click|preventDefault`, which assumes that `e.preventDefault();` was outside the `if` statement like
```js
 e.preventDefault();
if (!cookiesAreEnabled()) {
  ...
```

Could you confirm that this difference in logic doesn't cause any unexpected bahviour?

Reply:
it works fine from testing!

Reply:
I reverted it back, just in case it doesn't fully work!